types/mysql/mysql-tests.ts

```diff
@@ -396,9 +422,10 @@ connection.query({
 });
 
 connection = mysql.createConnection("mysql://localhost/test?flags=-FOUND_ROWS");
-connection = mysql.createConnection({ debug: true });
-connection = mysql.createConnection({ debug: ['ComQueryPacket', 'RowDataPacket'] });
-
-var type: mysql.FieldType = mysql.FieldType.SHORT;
-var info: mysql.IFieldInfo;
+connection = mysql.createConnection({debug: true});
+connection = mysql.createConnection({debug: ['ComQueryPacket', 'RowDataPacket']});
+/* it lints good
```

Comment:
Why is this code commented out?
Speaking of linting, could you add a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`?